CohortsToCreate.csv

```diff
@@ -0,0 +1,3 @@
+﻿atlasId,atlasName,cohortId,name
```

Comment:
Is this file used elsewhere in the branch? I don't see it

Reply:
https://github.com/OHDSI/CohortDiagnostics/pull/388/commits/4c839b4f2d6ab264e51e6484e63e4fd2a00707f3